xbeep.cpp

```diff
@@ -131,17 +143,17 @@ EXPORT std::string libxbee::XBee::mode(void) {
 	return std::string(mode);
 }
 
-EXPORT void libxbee::XBee::setLogTarget(FILE *f) {
+EXPORT void libxbee::XBee::setLogTarget(FILE *f) const {
```

Comment:
What's the feeling here - admittedly it doesn't change things at the C++ class level, but does change things in the underlying C context... is `const` still appropriate? (I suspect yes)

Reply:
Technically yes but I know... it's complicated.

Reply:
Thanks for your responses... Two points of view, rephrased for the next time:

1. the C++ class doesn't actually modify itself in these functions, thus `const` is correct.
2. use of `const` is misleading, as these functions do in fact alter the state of a "_libxbee_" instance.

I think we'll leave it, and claim that we're "_technically correct_" / point 1.

Happy to review if people have strong feelings about point 2.